### PR TITLE
Prepare for 0.2.0 release

### PR DESCRIPTION
**Pull request by kira-lin**
We plan to release 0.2.0 soon. A checklist:
[x] update readme
[x] fix CI of the examples

### CODE REVIEW

**Comment by kira-lin** [comment]
Fix the seed of random_split. Hopefully this fixes the randomly occurring batchnorm issue.

File: .github/workflows/raydp.yml [hunk 1]
```diff
@@ -50,9 +50,11 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
+          pip install wheel
+          pip install pytest koalas tensorflow torch tabulate ray[tune] xgboost_ray
           HOROVOD_WITH_GLOO=1
           HOROVOD_WITH_PYTORCH=1
-          pip install wheel pytest koalas tensorflow torch tabulate ray[tune] xgboost_ray horovod[pytorch,ray]
+          pip install horovod[pytorch,ray]
```
Comment (ConeyLiu) [inline .github/workflows/raydp.yml:57]: what's the difference?

Comment (kira-lin) [reply .github/workflows/raydp.yml:57]: install wheel first so that for the rest pip can build their wheels. For horovod I'm not sure, just because once in the CI it says horovod torch is not built. So I try to install it after pytorch is installed. And the running time of this step does vary after this change.

Comment (ConeyLiu) [reply .github/workflows/raydp.yml:57]: OK, thanks.
